addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1825,6 +1825,9 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
             this.colorPaletteEl.remove();
             this.colorPaletteEl = newEl;
         }
+        if (this.colorPaletteEl.closest('.o_we_so_color_palette').getAttribute('data-css-property') === 'background-color') {
+            options.getEditableCustomColors = wysiwyg.colorPalettesProps?.background?.getEditableCustomColors;
```

Comment:
A few line above this code, there is a `if (wysiwyg) {`. I am not actually sure it is required to check that wysiwyg exist but in doubt, you could add this line over there?